test/arena_mt.c

```diff
@@ -52,6 +54,9 @@ run(void *p __attribute__((unused)))
 void
 bench(int count)
 {
+	plan(0);
+	header();
```

Comment:
Please don't use `plan(0)`. Instead replace `fail` with `ok` or `is` where appropriate.

Reply:
Done. For some tests (like random or very extensive tests) replacing `fail` with `ok` does not look right. In this case if there are no `ok`-ish tests then `plan(1)` and `ok(true)` is added.